### ISSUE TITLE
Export wallet details JS error in current `master` branch

### ISSUE THREAD

**Issue by kdmukai** [cryptoadvance/specter-desktop#1737]
**Describe the bug**
Javascript console error:
```
Uncaught TypeError: Failed to execute 'appendChild' on 'Node': parameter 1 is not of type 'Node'.
    at showPageOverlay ((index):175:63)
    at HTMLButtonElement.onclick ((index):2588:102)
```

**To Reproduce**
1. Navigate to a wallet (multisig in my case)
2. Click on its Settings tab
3. Click on the Export sub tab
4. Click on "Go to export details"

Nothing happens. If you open a console window, you'll see the JS error pasted above.

**Expected behavior**
Should open the wallet details page/overlay.

**Screenshots**
<img width="1520" alt="Screen Shot 2022-05-28 at 8 32 47 AM" src="https://user-images.githubusercontent.com/934746/170827915-0ee0b2df-6d4a-49a5-9236-7c0af9dfed44.png">

**Desktop (please complete the following information):**
 - Where is your node running: local regtest
 - Where is your specter running: local from current (2022-05-28) `master` branch
 - If you're running a binary, which OS: Not a binary, but on macOS
 - Browser: Chrome 101.0.4951.64
 - Specter Version: n/a (current `master`)

**Comment by moneymanolis** [comment]
Can confirm the bug, as with #1738, the bug appears to only exist with multisig wallets.

**Comment by moneymanolis** [comment]
This was also fixed with #1741 (fixing the misplaced jinja tag).
